src/Microsoft.AspNetCore.Blazor/Json/CamelCase.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Blazor.Json
+{
+    internal static class CamelCase
+    {
+        public static string ToCamelCase(this string value)
+        {
+            // Basic cases: if we don't need to modify the value, bail out without creating a char array
+            if (string.IsNullOrEmpty(value))
+            {
+                return value;
+            }
```

Comment:
Is this considered invalid input since only we're calling this, and only for valid member names?

Reply:
Hmm. Debatable.

If we were to reject `""`, then I think we'd at least have to rename this to `CamelCaseDotNetMemberName` because it couldn't call itself a general camelcaser. The existing implementation, even without doing anything about spaces, I think fits a reasonable definition of a general camelcase implementation.

Since it's internal anyway I'd rather not get hung up on this. Let me know if you feel strongly about it.

Reply:
OK on more consideration I will be more explicit about limiting the scope to .NET identifiers and will rename the method accordingly.

Reply:
Renamed to `MemberNameToCamelCase`